tools/scripts/Makefile

```diff
@@ -0,0 +1,265 @@
+export PROCESSOR_CORE 
+export HARDWARE
+export PLATFORM
+
+NOOS_DIR 	  = $(CURDIR)/../..
+ROOT_DIR 	  = ../..
```

Comment:
doer ROOT_DIR get used?
if yes, is ROOT_DIR == NOOS_DIR?

Reply:
Removed ROOT_DIR. Thanks